nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/rendering/impl/RenderingServiceImpl.java

```diff
@@ -47,7 +46,7 @@
 
     public static final String EP_RENDER_ENGINES = "engines";
 
-    private final Map<String, RenderingEngine> engines = new HashMap<>();
```

Comment:
when i made my tests i had some problems with this class where in some case my tests failed because the needed template is overdid, then i changed the `Map#Key` by the name of the template... and i update to made it thread safe. Perhaps as i said  https://github.com/nuxeo/nuxeo/pull/3623#discussion_r360949205 i will change the commit message or i will dispatch into deux different commits

Reply:
for more readability  i changed the commit message to: ` NXP-28253: Fix template overriding on notification rendering`